### DISCUSSION TITLE
Request for concise production installation instruction on Linux systems

### DISCUSSION THREAD

**Discussion by maxxentropy** [datalust/seq-tickets#1739] [Help]
Is there anybody in the community or the dev team that has installation instructions for installing the docker image on a Linux system where the host is available on the Internet ( not just set up for localhost access )

I have tried to follow the directions available on both the Datalust sight and the docker hub sight, but the container is clearly not configured correctly. While it runs, it is not accessible from the URL of the server.

1) How do you specify a domain URL mapping to the seq containers UI/Ingestion ports?
the directions imply the -p 443:443 maps from localhost:<port> to seq port.
2) Is moving the domains' SSL certs to the <mount volume>/data/Certificates directory enough to ensure the seq containers seq server will use them?
3) The directions for working with the containers seq instance directly are challenging to follow. ( is it even reasonable to modify the Seq.json directly instead of having to pass all the config parameters to the run command.) 

I really love this tool and would like to be able to use it in the Linux environment but so far I am stumped.

Thanks
-Sean

**Comment by liammclennan** [comment]
Hi Sean,

I'll try to answer each of your questions.

1. TCP ports are independent of URLs. The server has only one TCP port 443, so `-p 443:443` means the host's TCP 443 port is mapped to the container's TCP 443 port, regardless of which domain names are mapped to the server's IP address. 
2. [Yes](https://docs.datalust.co/docs/https-tlsssl). The trick is that `/data` on the server is whatever host directory you have mapped with `-v`. E.g. If you start the container with `-v /seq:/data` then the certificate needs to go into `/seq/Certificates` on the host file system. 
3. The seq executable (on linux) is [run with docker](https://docs.datalust.co/docs/running-server-commands). Make sure you stop the Seq container first. E.g. 

```
docker run \
  -e ACCEPT_EULA=Y \
  -v $SEQ_DATA:/data \
  datalust/seq \
  show-key
```
